app/controllers/api/physical_servers_controller.rb

```diff
@@ -1,4 +1,50 @@
 module Api
   class PhysicalServersController < BaseController
+    def blink_loc_led_resource(type, id, _data)
+      change_resource_state(:blink_loc_led, type, id)
```

Comment:
@rodneyhbrown7 this is looking good! The only thing I want to suggest is that maybe here you could wrap these calls/responses in an asynchronous `api_action` like we do for other power operations (see https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api/vms_controller.rb#L11-L22 for an example of this), that is, if these operations are doing things over the network and are potentially costly in terms of response time.